bioframe/io/resources.py

```diff
@@ -117,13 +179,11 @@ def fetch_centromeres(db, provider=None, merge=True, verbose=False):
     - centromeres.txt
     - cytoBandIdeo
     - cytoBand
-    - gap.txt
-
+    
+    Gap files no longer provide centromere information.
```

Comment:
specify this is due to t2t? or not worth mentioning?

Reply:
I don't think it's due to T2T. I think even the hg38 gap files don't report them anymore.